src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -32,6 +32,8 @@ program_node::program_node(std::shared_ptr<primitive> prim, program& prog)
     : desc(prim), myprog(prog), org_id(prim->id) {
     if (prim)
         output_layout.data_padding = prim->output_padding;
+
+    primitive_impl_caches = std::make_shared<LRUCache<std::string, std::shared_ptr<primitive_impl>>>(4 * sizeof(std::shared_ptr<primitive_impl>));
```

Comment:
Lets make this "4" to be more understandable & controllable, i.e.g, declare it as a const variable at least.

Reply:
Fixed it.